D/DGGRID7/build_tarballs.jl

```diff
@@ -37,4 +41,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"8.1.0")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version=v"8.1.0", julia_compat="v1.6.0")
```

Comment:
To change the `julia_compat` you ***must*** change the version number: #2763

Reply:
`julia_compat="v1.6.0"` fixed in 9c5c12de1f46e09bfb63c41bd04847164d842e7f

Reply:
The version number of the packag, as explained in the non-negotiable requirements in issue #2763 linked above

Reply:
ok. See f9bf39bcf252f89420ec258038f477e0f143c2d5